lib/leadership.js

```diff
@@ -1,22 +1,66 @@
+module.exports = resolveLeadership;
```

Comment:
not really a fan of this style, my preference is to simply export symbols only, so that

import {resolveLeadership, BasicLeadershipElection} from "node-discover/leadership" 

works

With this style, its 

import * as resolveLeadership, {BasicLeadershipElection} from "node-discover/leadership" 

which is not clean IMO.

recommended to go back to module.exports = { ... }

Reply:
Are you using resolveLeadership from outside the module?

Reply:
Also, I exposed the default leadership modules on the node-discover constructor.

Can't you do:

```js
import { BasicLeadershipElection } from 'node-discover'
```

Reply:
I wasn't meaning as part of the public api, this wouldn't impact me, just my opinion on defining exports, I don't see value in this vs just exporting resolve as a symbol.

typically default exports like this is used for classes when the entire point of the file is the class. 

i'm in favor of dealing with exports in a way that is cleanly importable in module format.

Reply:
Also it's simpler for type definement too ie for typescript .d.ts files (this syntax is possible yeah, just not as 'clean' IMO)

Reply:
Thank you for your comments on this. I will take your feedback under advisement.